website/redirects.txt

```diff
@@ -0,0 +1,3 @@
+/get/macos/taq https://github.com/ecadlabs/taqueria/releases/latest/download/taq-macos 301
+/get/linux/taq https://github.com/ecadlabs/taqueria/releases/latest/download/taq-linux 301
+/get/windows/taq https://github.com/ecadlabs/taqueria/releases/latest/download/taq-windows.exe 301
```

Comment:
I believe @jevonearth mentioned to use temporary redirect. It should be `307` status code here.

Reply:
Done